docs/source/configuration/authn-jwt.mdx

```diff
@@ -134,8 +156,33 @@ fn process_request(request) {
 }
 ```
 
-Note: APOLLO_AUTHENTICATION_JWT_CLAIMS is a rhai scope constant with a value of `"apollo_authentication::JWT::claims"`.
+## Propagating JWTs to subgraphs
+
+Because clients provide their JWT via an HTTP header, you can propagate JWTs to some or all of your subgraphs using the Apollo Router's general-purpose [header propagation settings](./header-propagation).
+
+
+## Observability
+
+If your router enables [tracing](./tracing/), the JWT authentication plugin has its own tracing span, named `authentication plugin`.
```

Comment:
```suggestion
If your router enables [tracing](./tracing/), the JWT authentication plugin has its own tracing span, named `authentication_plugin`.
```
(After code review I had been advised to rename this to not include " ", so I did.)